src/Nethermind/Nethermind.Evm.Test/VirtualMachineTestsBase.cs

```diff
@@ -68,7 +68,7 @@ public virtual void Setup()
         TestState = new WorldState(trieStore, codeDb, logManager);
         _ethereumEcdsa = new EthereumEcdsa(SpecProvider.ChainId);
         IBlockhashProvider blockhashProvider = new TestBlockhashProvider(SpecProvider);
-        CodeInfoRepository = new CodeInfoRepository();
+        CodeInfoRepository = new CodeInfoRepository(1);
```

Comment:
Why are we passing `1` as `chainId` instead of `SpecProvider.ChainId`?

Reply:
wrong PR

Reply:
wrong PR